src/Tags/Locales.php

```diff
@@ -95,14 +97,16 @@ private function getLocale($key)
     private function addData($locales)
     {
         return $locales->map(function ($locale, $key) {
-            if (! $localized = $this->getLocalizedData($key)) {
-                return null;
+            $localized = $this->getLocalizedData($key);
+
+            if ($localized || $this->params->bool('include_all')) {
+                $localized['locale'] = $locale;
+                $localized['current'] = Site::current()->handle();
+                $localized['is_current'] = $key === Site::current()->handle();
+                $localized['available'] = Arr::exists($localized, 'status');
```

Comment:
Can you change `available` to `exists`? We use that in other places.

Reply:
Makes sense. Will change it.